.agent/rules/react-joyride-callback.md

```
@@ -0,0 +1,106 @@
+
+  return (
+    <div>
+      <Joyride callback={handleJoyrideCallback} run={run} stepIndex={stepIndex} steps={steps} />
+      <button onClick={handleClickStart}>Start</button>
+      // Your code here...
+    </div>
+  );
+}
+```
+
+You can read more about the constants [here](https://docs.react-joyride.com/constants)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)